src/main/resources/com/owlcyberdefense/cobol1/xsd/cobol1.dfdl.xsd

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns="http://www.w3.org/2001/XMLSchema"
+    xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" 
+    targetNamespace="http://example.com"
+    xmlns:tns="http://example.com"
+    xmlns:ex="http://example.com"
+    xmlns:fn="http://www.w3.org/2005/xpath-functions">
+
+  <include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+
+  <element name="RECORDS" dfdl:lengthKind="implicit">
+    <complexType>
+      <sequence >
+        <element maxOccurs="unbounded" ref="tns:MY-RECORD"/>
+      </sequence>
+    </complexType>
+  </element>
+
+  <annotation>
+    <appinfo source="http://www.ogf.org/dfdl/">
+      <dfdl:defineFormat name="cobol1Format">
+        <dfdl:format ref="tns:GeneralFormatPortable"
+          alignment="1" 
+          alignmentUnits="bytes" 
+          binaryBooleanFalseRep="0" 
+          binaryBooleanTrueRep="1" 
+          binaryDecimalVirtualPoint="0" 
+          binaryFloatRep="ieee" 
+          binaryNumberCheckPolicy="strict" 
+          binaryPackedSignCodes="C D F C" 
+          byteOrder="bigEndian" 
+          calendarCenturyStart="53" 
+          calendarCheckPolicy="strict" 
+          calendarFirstDayOfWeek="Monday"           
+          calendarLanguage="en-US" 
+          calendarObserveDST="yes" 
+          calendarTimeZone="" 
+          choiceLengthKind="implicit" 
+          decimalSigned="yes" 
+          encoding="ebcdic-cp-us"  
+          fillByte="%#r00;" 
+          lengthKind="explicit" 
+          lengthUnits="bytes" 
+          occursCountKind="implicit" 
+          representation="binary"  
+          textBooleanFalseRep="%#r01;" 
+          textBooleanJustification="left" 
+          textBooleanPadCharacter="%SP;" 
+          textBooleanTrueRep="%#r00;" 
+          textCalendarJustification="left" 
+          textCalendarPadCharacter="0"
+          textNumberJustification="right" 
+          textNumberPadCharacter="0" 
+          textNumberRep="zoned" 
+          textNumberRounding="pattern" 
+          textNumberRoundingMode="roundUp" 
+          textPadKind="none" 
+          textStandardBase="10"
+          textStringJustification="left" 
+          textStringPadCharacter="%SP;" 
+          textTrimKind="none" 
+          textZonedSignStyle="asciiStandard"  
+          >
+        </dfdl:format>
+      </dfdl:defineFormat>
+      <dfdl:format ref="tns:cobol1Format"/>
```

Comment:
Consider removing any redundantly defined attribute definitions which are already defined in "GeneralFormatPortable" with the same values. 

Consider removing or replacing the "1" with a more meaningful abbreviation and flattening all files to just src and test.